packages/web/src/pages/collection-page/CollectionPageProvider.tsx

```diff
@@ -557,11 +562,13 @@ class CollectionPage extends Component<
         })
       )
     } else if (entries.length > 0) {
-      play(entries[0].uid, { isPreview })
+      const isOwner = collection?.playlist_owner_id === userId
+      stop()
+      play(entries[0].uid, { isPreview: isPreview && isOwner })
```

Comment:
i wonder if there's a cleaner way to write this now.. but no need for now if this works